cocos/network/WebSocket.h

```diff
@@ -212,6 +212,9 @@ class CC_DLL WebSocket
      */
     State getReadyState();
 
+    inline const std::string& getUrl() { return _url; }
+    inline const std::string& getProtocol() { return _selectedProtocol; }
```

Comment:
please add const like
```c++
inline const std::string& getProtocol() const
```
i am not sure if inline and const can work together or not